program-bpf-rust/src/program_command.rs

```diff
@@ -23,36 +24,48 @@ mod test {
         let cmd = Command::InitDashboard;
         let mut b = vec![0; 16];
         cmd.serialize(&mut b).unwrap();
-        assert_eq!(b, [0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]);
+        assert_eq!(b[0..4], [0, 0, 0, 0]);
 
         let cmd = Command::InitPlayer;
         let mut b = vec![0; 16];
         cmd.serialize(&mut b).unwrap();
-        assert_eq!(b, [1, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]);
+        assert_eq!(b[0..4], [1, 0, 0, 0]);
 
         let cmd = Command::InitGame;
         let mut b = vec![0; 16];
         cmd.serialize(&mut b).unwrap();
-        assert_eq!(b, [2, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]);
+        assert_eq!(b[0..4], [2, 0, 0, 0]);
 
         let cmd = Command::Advertise;
         let mut b = vec![0; 16];
         cmd.serialize(&mut b).unwrap();
-        assert_eq!(b, [3, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]);
+        assert_eq!(b[0..4], [3, 0, 0, 0]);
 
         let cmd = Command::Join;
         let mut b = vec![0; 16];
         cmd.serialize(&mut b).unwrap();
-        assert_eq!(b, [4, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]);
+        assert_eq!(b[0..4], [4, 0, 0, 0]);
 
         let cmd = Command::KeepAlive;
         let mut b = vec![0; 16];
         cmd.serialize(&mut b).unwrap();
-        assert_eq!(b, [5, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]);
+        assert_eq!(b[0..4], [5, 0, 0, 0]);
 
         let cmd = Command::Move(1, 2);
         let mut b = vec![0; 16];
         cmd.serialize(&mut b).unwrap();
         assert_eq!(b, [6, 0, 0, 0, 1, 2, 0, 0, 0, 0, 0, 0, 0, 0, 0, 0]);
     }
+
+    #[test]
+    fn pull_in_externs() {
+        // Rust on Linux excludes the solana_sdk_bpf_test library unless there is a
+        // direct dependency, use this test to force the pull in of the library.
+        // This is not necessary on macos and unfortunate on Linux
+        // Issue #4972
```

Comment:
nit: reference the repo path to this issue, since it doesn't exist in the example-tictacto repo